app/views/tasks/_task.html.erb

```diff
@@ -2,4 +2,14 @@
   <h2><%= task.title %></h1>
   <p><%= task.description %></p>
   <p><%= task.date_and_time %></p>
+  <% if task.complete? %>
+    <p class="completed-task"><strong>Completed</strong></p>
+  <% else %>
+    <p class="uncompleted-task"><strong>Not completed</strong></p>
+    <%= form_with(url: tasks_task_complete_url, html: {method: :post}, local: true) do |f|%>
```

Comment:
This is what I was talking about on the action name. task_task_complete is not a good helper name. Fix it. Also, _url should be used only inside the controllers, on the views please use _path